ocs_ci/utility/ibmcloud.py

```diff
@@ -520,6 +522,268 @@ def delete_volume_id(self, volume):
+                sample = TimeoutSampler(
+                    timeout=timeout,
+                    sleep=10,
+                    func=self.check_node_status,
+                    node_name=node.name,
+                )
+                sample.wait_for_func_status(result=constants.STATUS_STOPPED)
+        logger.info(f"Starting instances {list(nodes)}")
+
+        self.start_nodes(nodes=nodes)
+        if wait:
+            for node in nodes:
+                sample = TimeoutSampler(
+                    timeout=timeout,
+                    sleep=10,
+                    func=self.check_node_status,
+                    node_name=node.name,
+                )
+                sample.wait_for_func_status(result=constants.STATUS_RUNNING.lower())
+
+    def check_node_status(self, node_name):
+        """
+        Check the node status in IBM cloud
+        Args:
+            node_name (str): Node name
+        Returns:
+            str: Status of node
+        """
+        try:
+            cmd = f"ibmcloud is instance {node_name} --output json"
+
+            out = run_ibmcloud_cmd(cmd)
+            out = json.loads(out)
+            return out["status"]
+        except CommandFailed as cf:
+            if "Instance not found" in str(cf):
+                return True
+        return False
+
+    def restart_nodes_by_stop_and_start_force(self):
+        """
+        Make sure all nodes are up by the end of the test on IBM Cloud.
+        """
+        resource_name = None
+        stop_node_list = []
+        stopping_node_list = []
+        cmd = "ibmcloud is ins --all-resource-groups --output json"
+        out = run_ibmcloud_cmd(cmd)
+        all_resource_grp = json.loads(out)
+        cluster_name = config.ENV_DATA["cluster_name"]
+        for resource_name in all_resource_grp:
+            if cluster_name in resource_name["resource_group"]["name"]:
+                resource_name = resource_name["resource_group"]["name"]
+                break
+        assert resource_name, "Resource Not found"
+        cmd = f"ibmcloud is ins --resource-group-name {resource_name} --output json"
+        out = run_ibmcloud_cmd(cmd)
+        all_instance_output = json.loads(out)
+        for instance_name in all_instance_output:
+            if instance_name["status"] == constants.STATUS_STOPPED:
+                node_obj = OCP(kind="Node", resource_name=instance_name["name"]).get()
+                node_obj_ocs = OCS(**node_obj)
+                stop_node_list.append(node_obj_ocs)
+            if instance_name["status"] == constants.STATUS_STOPPED:
+                node_obj = OCP(kind="Node", resource_name=instance_name["name"]).get()
+                node_obj_ocs = OCS(**node_obj)
+                stopping_node_list.append(node_obj_ocs)
+                stop_node_list.append(node_obj_ocs)
+        logger.info("Force stopping node which are in stopping state")
+        self.stop_nodes(nodes=stop_node_list, force=True, wait=True)
```

Comment:
I think it should be `self.stop_nodes(nodes=stopping_node_list, force=True, wait=True)`

Reply:
i have update the code